larq_compute_engine/tests/BUILD

```diff
@@ -29,6 +29,14 @@ py_test(
     ],
 )
 
+py_test(
```

Comment:
I used `py_test` instead of `py_binary` in order to enable running this via `bazel test` which makes sure that the same compile flags are set so local build caches don't get nuked.